libs/template/presets/storefront/experience/header.ts

```diff
@@ -13,17 +14,29 @@ export const HeaderTemplate: ExperienceComponent = {
         {
           type: 'oryx-content-link',
           content: { data: { text: 'FREE DELIVERY & RETURNS' } },
-          options: { url: '/', icon: IconTypes.Check },
+          options: {
```

Comment:
All these changes should be done using feature flags.

Reply:
as discussed we make the exception here because of this is considered a fix.